newIDE/app/src/UI/Theme/DefaultTheme/index.js

```diff
@@ -173,6 +173,12 @@ const muiTheme = createMuiTheme({
         minHeight: 32, // Reduce the height of tabs to 32px
       },
     },
+    // Use a more visible color scheme for tabs:
+    MuiAutocomplete: {
+      root: {
+        alignSelf: "center",
```

Comment:
Add a comment explaining why this is used, especially because MuiAutocomplete is used for other stuff (autocomplete for objects list in the events sheet, in the command palette, etc...).

Reply:
Oh yes the command palette, it is completely broken with this hack.
Instead i can add a style in the resourceSelector from the text Object porperties like your suggest, it need pass props for that.
I understand why fix this issue wasn't yet fixed, when we fix something, somewhere else a thing broke, it's endless 😅

Reply:
Yeah that's why I heavily favor the approach "different CSS style for each component" and try to minimize the global CSS variables :) Sounds good now with just a style in the `ResourcesSelector`